doc/developer-guides/hld/hv-hypercall.rst

```diff
@@ -9,6 +9,28 @@ is from the hypervisor to the guest VM. The hypervisor supports
 hypercall APIs for VM management, I/O request distribution, interrupt injection,
 PCI assignment, guest memory mapping, power management, and secure world switch.
 
+The application binary interface (ABI) of ACRN hypercalls is defined as follows.
+
+- A guest VM executes the ``vmcall`` instruction to trigger a hypercall.
+
+- Input parameters of a hypercall includes:
+
+  - An hypercall ID in ``R8``, which specifies the kind of service requested by
+    the guest VM.
```

Comment:
What is R8, RDI, RSI? Are they registers? If so, should we say that they are registers for the sake of clarity? (Or would the audience know that?)

Reply:
Yes. They are registers. Will explicitly state so in the next revision.